subsys/bluetooth/audio/bap_scan_delegator.c

```diff
@@ -1170,10 +1170,58 @@ BT_GATT_SERVICE_DEFINE(bass_svc,
 	RECEIVE_STATE_CHARACTERISTIC(2)
 #endif /* CONFIG_BT_BAP_SCAN_DELEGATOR_RECV_STATE_COUNT > 2 */
 #endif /* CONFIG_BT_BAP_SCAN_DELEGATOR_RECV_STATE_COUNT > 1 */
-);
+};
+
+static struct bt_gatt_service bass_svc = BT_GATT_SERVICE(attr_bass_svc);
+
+static bool scan_delegator_registered;
+static bool pa_sync_cb_registered;
+
+static int bt_bap_bass_register(void)
+{
+	int err;
+
+	err = bt_gatt_service_register(&bass_svc);
+	if (err) {
+		LOG_DBG("Failed to register BASS service (err %d)", err);
+		return err;
+	}
+
+	LOG_DBG("BASS service registered");
+
+	return 0;
+}
+
+static int bt_bap_bass_unregister(void)
```

Comment:
```suggestion
static int bass_register(void)
{
	int err;

	err = bt_gatt_service_register(&bass_svc);
	if (err) {
		LOG_DBG("Failed to register BASS service (err %d)", err);
		return err;
	}

	LOG_DBG("BASS service registered");

	return 0;
}

static int bass_unregister(void)
```

Suggest to omit the `bt_bap` prefix for `static functions

Reply:
Done.